contracts/rate_oracles/CompoundBorrowRateOracle.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity =0.8.9;
+
+import "../interfaces/rate_oracles/ICompoundRateOracle.sol";
+import "../interfaces/compound/ICToken.sol";
+import "./BaseRateOracle.sol";
+import "../interfaces/compound/InterestRateModel.sol";
+
+contract CompoundBorrowRateOracle is BaseRateOracle, ICompoundRateOracle {
+    /// @inheritdoc ICompoundRateOracle
+    ICToken public immutable override ctoken;
+
+    /// @inheritdoc ICompoundRateOracle
+    uint8 public immutable override decimals;
+
+    uint8 public constant override UNDERLYING_YIELD_BEARING_PROTOCOL_ID = 2; // id of compound is 2
```

Comment:
the id should be 6 for the compound borrow rate oracle since 2 is reserved for the compound supply rate oracle